src/serlo-editor/store/root/saga.ts

```diff
@@ -7,7 +7,7 @@ import { selectDocuments } from '../documents'
 import { handleRecursiveInserts } from '../documents/saga'
 import { persistHistory } from '../history'
 
-// TODO: Move initRootSaga to documents slice and remove root slice completely
+//TODO: Move initRootSaga to documents slice and remove root slice completely
```

Comment:
@hejtful should we keep this TODO or turn it into an issue / normal comment?

Reply:
It can be a normal comment, and milder wording, like "Move..." -> "Consider moving..."